ddtrace/commands/ddtrace_run.py

```diff
@@ -105,6 +104,9 @@ def main():
 
     log.debug("program executable: %s", executable)
 
+    if "uwsgi" in executable:
```

Comment:
maybe make that a little more robust?

Reply:
well, I suppose a more precise check like `executable.endswith("uwsgi")` but did you mean something different here with "robust"?

Reply:
Yeah, I like the heuristic but if somebody has a program named `iloveuwsgi` it'll match… Not sure if `== "uwsgi"` is too strict?

Reply:
Good point, I replaced it with more strict equality.